documentation/backup.md

```diff
@@ -0,0 +1,49 @@
+# Backup Puppet Enterprise using the PEADM module
+
+## What is being backed up?
+
+By default the `peadm::backup` plan will backup the following items:
+
+1. Orchestrator
+2. PuppetDB
+3. RBAC
+   - Also, It copies the LDAP secret key if it present
+4. Activity
+5. Classification
+
+Optionally you can also backup:
+
+1. CA (CA and SSL certificates)
+
+**Note:** It is important to note that the `peadm::backup` plan in its current version is not as granular as you can get when you manually run [the `puppet-backup create` command.](https://puppet.com/docs/pe/2021.7/backing_up_and_restoring_pe.html#back_up_pe_infrastructure)
+
+## How can I customize my backup?
+
+We need to pass the `backup` parameter to the `peadm::backup` plan.
```

Comment:
The backup parameter is of type `Peadm::Recovery_opts` - that is unclear to the user. Also, the name `backup` as parameter choice is somewhat ambiguous given the name of the plan itself, what about `backup_options` or `include_options`?
Can we make sure the plan documents the options so `bolt plan show` already explains what the possibilities are?
It seems we always need to create a params json file, is it an idea to offer an additional, simplified CLI?

Reply:
Marking this as **feature-request**